Learn-CSS-Blog/index6.html

```diff
@@ -0,0 +1,20 @@
+<!DOCTYPE html>
+<html>
+<head>
+    <meta charset="UTF-8">
+    <title>Home | Learn Basic CSS by building a Blog</title>
+</head>
+<body>
+    <div></div>
+</body>
+</html>
+
+<!-- 
+    Give the `<div> an `id` attribute with the value `main`.
```

Comment:
Give the `<div> an `id` attribute with the value "main".
    Here is an example:
     ```
    <div id="wrapper">
    </div>
    ```